src/Tizen.NUI/src/internal/Interop/Interop.GLWindow.cs

```diff
@@ -110,6 +110,12 @@ internal static partial class GLWindow
             [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_GlWindow_RenderOnce")]
             public static extern void GlWindowRenderOnce(HandleRef jarg1);
 
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_GlWindow_GetRenderingMode")]
```

Comment:
could you update the csharp-binder's patch on this PR? and csharp-binder patch should be merged prior to this PR. otherwise there will be a runtime exception which blocks the release of tizenfx by TCT hurdle

Reply:
I see. csharp-binder's patch is only devel/master. The patch is not merged yet to tizen branch.
After merged, I will try to upload again.
Thank you for your detail review.